### PR TITLE
Vb 1814 delete duplicate booking in production database

### PR DESCRIPTION
**Pull request by AledWEvans**
## What does this pull request do?

deletes a duplicate booking

## What is the intent behind these changes?

removed unwanted data and unwanted bookings

### CODE REVIEW

File: src/main/resources/db.scripts.mvp/delete_duplicate_booking.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    SET SCHEMA 'public';
+
+    CREATE TEMP TABLE tmp_visit_ids_to_be_deleted(
+        visit_id         int
+    );
+
+    --selection criteria for visits to be deleted
+    INSERT INTO tmp_visit_ids_to_be_deleted (visit_id) (
+        select v.id from visit v
+                join legacy_data ld on ld.visit_id = v.id
```
Comment (dhirajshettymoj) [inline src/main/resources/db.scripts.mvp/delete_duplicate_booking.sql:12]: Not a legacy visit I presume.

Comment (AledWEvans) [reply src/main/resources/db.scripts.mvp/delete_duplicate_booking.sql:12]: I thought that but It was a legacy visit

Comment (dhirajshettymoj) [reply src/main/resources/db.scripts.mvp/delete_duplicate_booking.sql:12]: oh ok.. Will approve the change then

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request merged**